src/test/java/uk/gov/hmcts/ccd/domain/service/aggregated/DefaultGetCriteriaOperationTest.java

```diff
@@ -24,14 +27,14 @@
 import org.mockito.Mock;
 import org.mockito.MockitoAnnotations;
 
-class DefaultFindSearchInputOperationTest {
+public class DefaultGetCriteriaOperationTest {
     @Mock
     private UIDefinitionRepository uiDefinitionRepository;
     @Mock
     private CaseDefinitionRepository caseDefinitionRepository;
 
     private final CaseType caseType = new CaseType();
-    private DefaultFindSearchInputOperation findSearchInputOperation;
+    private DefaultGetCriteriaOperation defaultGetCriteriaOperation;
```

Comment:
final fields should all appear before non-final ones.

Reply:
applied for this one, but better put it in a check style rule for consistency